plugins/modules/sonic_system.py

```diff
@@ -75,6 +75,13 @@
             description:
               - Specifies the mac anycast-address
             type: str
+      auto_breakout:
+        description:
+          - Specifies auto-breakout status in the device
+        type: str
+        choices:
+          - ENABLE
+          - DISABLE
```

Comment:
The following "EXAMPLES" section should be added with some examples for auto_breakout, which should come from the resource model examples files.

Reply:
> The following "EXAMPLES" section should be added with some examples for auto_breakout, which should come from the resource model examples files.

Added examples in the file.